config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatasets.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -77,18 +77,15 @@ spec:
                           description: The dataset this entry applies to.
                           properties:
                             datasetId:
-                              description: Required. A unique ID for this dataset,
-                                without the project name. The ID must contain only
+                              description: Required. A unique Id for this dataset,
```

Comment:
Maybe not for this PR. This feels like a regression. I believe we want "ID" as opposed to "Id"?
@yuwenma